exploratory_analysis/Crypto_Microstructure_BTC_USDT_Exploratory_Analysis.ipynb

```diff
@@ -0,0 +1,4634 @@
+          },
+          "metadata": {
+            "needs_background": "light"
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The scatter here is flat too. Same picture with L1 data where ratio number below 1 seems to result in more volatile returns in the next period. But notice I have the limiter in place for x-axis. The crypto market doesn't have circuit breaker (well unless you count exchange servers going down in the heat of frantic days)."
+      ],
+      "metadata": {
+        "id": "RZ935GAICG2t"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "fig, ax = plt.subplots()\n",
+        "ax.scatter(x = l2['ratio'], y = l2['log_returns'])\n",
+        "slope, intercept = np.polyfit(l2['ratio'], l2['log_returns'], 1)\n",
+        "plt.plot(l2['ratio'], slope * l2['ratio'] + intercept, color = 'red')"
```

Comment:
using the symmetric order book imbalance instead of ratio would help here a lot too!